src/RequestForm.js

```diff
@@ -381,6 +381,11 @@ class RequestForm extends React.Component {
         if ((blocks.length && blocks[0].userId === selectedUser.id) || (!isEmpty(automatedPatronBlocks) && !isAutomatedPatronBlocksRequestInPendingState)) {
           state.blocked = true;
           state.isPatronBlocksOverridden = false;
+
+          this.setState({
```

Comment:
@annamelnyk this looks good but I'm wondering do we still need line 382 and 383 or could we remove it?

````js
state.blocked = true;
state.isPatronBlocksOverridden = false;
````

Reply:
Thanks, removed it